pkgs/applications/science/logic/aspino/default.nix

```diff
@@ -1,34 +1,48 @@
-{ stdenv, fetchFromGitHub, zlib, boost, glucose }:
+{ stdenv, fetchurl, fetchFromGitHub, zlib, boost, glucose }:
+
+let
+  glucose' = glucose.overrideAttrs (_: {
```

Comment:
We don't even need to build glucose here, we just need the sources.  Please remove the override and just use `glucoseSrc = fetchurl ...`.

It's unfortunate that upstream did not manage to use versioned tarballs, but there's nothing we can do about that.

Reply:
ahh, you're absolutely right, will fix this.

> It's unfortunate that upstream did not manage to use versioned tarballs, but there's nothing we can do about that.

this is one of the reasons why I added myself as maintainer as well, so in case of any breackage in the future I can help out :)